providers/apis/dydx/parse.go

```diff
@@ -0,0 +1,216 @@
+package dydx
```

Comment:
Moved most of the previous functionality into this file just to keep em separate.